docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+
+6. The updated chatbot with LangChain is built as a container and deployed to OpenShift.
+
+## Requirements
+It is expected that you have admin access to an OpenShift 4.12+ cluster. The following code was tested with an OpenShift 4.15 cluster and OpenShift AI 2.9.
+
+## Podman AI Lab
+### Install Podman Desktop and Podman AI Lab extension
+Follow the installation instructions for Podman Desktop and the Podman AI Lab extension in the below Red Hat Developer article. The article also gives a great overview of the features in Podman AI Lab. 
+[Podman AI Lab - Getting Started](https://developers.redhat.com/articles/2024/05/07/podman-ai-lab-getting-started){:target="_blank"}
+
+### Download the model
+We will be downloading and using the TheBloke/Mistral-7B-Instruct-v0.2-GGUF. This model is quantised (smaller) version of the full Mistral-7B-Instruct-v0.2. The smaller model will allow us to run inferencing on CPUs if GPUs are not an option.
+
+
+1. Go to the AI Lab extension and select Catalog under Models.
+![Catalog](img/AI_lab_catalog.png)
+
+2. If you haven't already, download the _TheBloke/Mistral-7B-Instruct-v0.2-GGUF_ model.
+![Download Model](img/mistral_download.png) 
++ The model is around 4GB so it might take some time. 
+![Download Progress](img/mistral_download_progress.png)
+
+3. Podman AI Lab allows you to get started quickly with downloaded models through Services, Playgrounds, and the Recipes Catalog. 
+      1. **Services** -
+      The Services section allows you to create a model service endpoint for models you've downloaded. Client code is provided (cURL by default) in multiple formats to get you started quickly with sending in requests to the model service endpoint. 
+      ![Podman AI Lab Services](img/podman_AI_services.png)
+
+      1. **Playgrounds** -
+      The Playgrounds area allows you to define system prompts and experiment with different settings like temperature, max tokens, and top-p.
+      ![Podman AI Lab Playgrounds](img/podman_AI_playground.png)
+
+      1. **Recipes Catalog** -
+      The Recipes Catalog contains demo applications for Natural Language Processing (NLP), Computer Vision, and Audio. We'll be using the ChatBot recipe demo code in this example.
+
+5. **Create the Chatbot** -
+Make sure to select _TheBloke/Mistral-7B-Instruct-v0.2-GGUF_ as your model and then click **_Start AI App_** button.
+![Start Chatbot](img/start_chatbot_recipe.png)
+
+6. After the chatbot has started open it up to test it out.
+![Chatbot UI](img/chatbot_recipe_UI.png)
+
+7. At the bottom of the AI App Details section you'll see a **_Open in VSCode_** button. Clicking on that will open all of the code that is running your chatbot. Later we'll modify that code to connect langchain, TheBloke/Mistral-7B-Instruct-v0.2-GGUF model, and the Elasticsearch Vector Database. 
+![Podman AI Chatbot Recipe](img/podman_AI_chatbot.png)
+
+## Deploying OpenShift AI
```

Comment:
This section should be optional, or point to OpenShift AI installation instructions. Most people won't start from a vanilla OpenShift cluster.

Reply:
Made the section optional, included the installation instructions in the product documentation, and made the automated installation collapsible.